### ISSUE TITLE
Prevent empty headers in tables

### ISSUE THREAD

**Issue by dariavladykina** [tbazant/ai-prompts#3]
Another accessibility issue can happen in tables that lack one or more headers. For more information, see https://github.com/SUSE/doc-styleguide/issues/312.

Prompt suggestion:
If a table is missing a row heading and/or column heading, add it according to the table content. 

Vale suggestion:
Create a warning when the table is missing a row heading and/or column heading.
OR:
Create a warning when the element `thead` that marks up the table head is missing AND/OR when the number of columns in `tgroup` cols does not match the number of rows under the `thead` element. 

Example:

```
<tgroup cols="2">
  <thead>
   <row>
     <entry>Maximum File Size</entry>```      -   only one row instead of 2!
``` </row>
  </thead> 
```

**Comment by tbazant** [comment]
Fixed by https://github.com/tbazant/ai-prompts/blob/51f6601ef21dbd800388cb71e16d790e5f081243/common/add-missing-table-headers.md
